core/commands/storage.go

```diff
@@ -52,9 +53,13 @@ const (
 	testOnlyOptionName            = "host-search-local"
 	storageLengthOptionName       = "storage-length"
 	repairModeOptionName          = "repair-mode"
+	payoutScheduleOptionName      = "payout-schedule"
+	numPayoutsOptionName          = "num-payouts"
 
-	defaultRepFactor     = 3
-	defaultStorageLength = 30
+	defaultRepFactor      = 3
+	defaultStorageLength  = 30
+	defaultPayoutSchedule = 0
+	defaultNumPayouts     = 1
```

Comment:
1 payout by default??

Reply:
as jin mentioned, you need to use constants, and also update go-btfs-common to latest release